Bizyhood/Utility.php

```diff
@@ -90,6 +90,9 @@ public static function buildDateText($start, $end, $single, $plural)
       
       $dates = '';
       
+      // set bizyhood timezone
+      date_default_timezone_set('America/Chicago');
```

Comment:
The timezone could be completely different depending on where the publisher is. We can't assume America/Chicago. Can you use the offset provided in the returned timestamp?